spec/src/main/asciidoc/microprofile-opentracing-spec.asciidoc

```diff
@@ -138,10 +138,10 @@ Spans created for incoming requests will have the following tags added by defaul
 * Tags.HTTP_STATUS
 * Tags.ERROR (if true)
 
-Tags.SPAN_KIND has to be specified at Span start time.
+Tags.SPAN_KIND MUST be specified at Span start time.
 
-An Tags.ERROR tag is added to a Span on failed operations. It means for any server error (5xx) codes.
-If there is an exception object available the implementation should also add logs `event=error` and `error.object=<error object instance>` to the active span.
+An Tags.ERROR tag SHOULD be added to a Span on failed operations. It means for any server error (5xx) codes.
```

Comment:
Are we not concerned with 4xx codes, because we expect the client to have added any info about those?

Reply:
+1 I will update the client section.


cc @objectiser